manage.py

```diff
@@ -251,6 +250,32 @@ def demo_db():
     for i in range(5):
         add_passover(timestamp=now + timedelta(minutes=i*10))
 
+    print("Database has been seeded with demo data.")
+
+
+def import_commands():
+    """Imports commands from the CommandDocs.txt file in ex2_ground_station_software.
+    """
+    try:
```

Comment:
Before opening the `CommandDocs.txt` file lets run the command to regenerate it in order to ensure that we are using the most up to date list of commands.

Reply:
In the comm_integration_testing branch of ex2_ground_station_software, I have a GitHub action setup so that CommandDocs.txt is updated whenever system.py (which contains all the commands but in Python) is updated. 

Should I keep this or have the website manage CommandDocs.txt generation?

Reply:
Yea no keep that there, thats great. Regenerating the `CommandDocs.txt` before importing the command is more of just a precautionary measure.